app/mailers/instructor_alert_mailer.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class InstructorAlertMailer < ApplicationMailer
+  def email(alert)
+    # return unless Features.email?
+    @alert = alert
+    set_course_and_creators
+    if @instructors.present?
```

Comment:
This is better as a guard clause: `return unless @instructors.any?` or `return if @instructors.empty?`

Reply:
sure